tests/test_language.py

```diff
@@ -19,6 +19,18 @@ def test_genexp(e):
     assert e('list(a + 1 for a in [1,2,3])') == [2, 3, 4]
 
 
+def test_bitwise_operators(e):
+    assert e('4 << 1') == 8
+
+    assert e('8 >> 1') == 4
+
+    assert e('4 | 1') == 5
+
+    assert e('5 ^ 6') == 3
+
+    assert e('3 & 6') == 2
```

Comment:
There was some discussion about reorganizing the simpleeval tests into the current system and files, potential later PR.
This section would likely be merged in with the other basic operators.